CI/harvester-installer/roles/jenkins/templates/config_jenkins_as_code.yaml.j2

```diff
@@ -12,14 +12,9 @@ jenkins:
   authorizationStrategy:
     globalMatrix:
       permissions:
-        - "Job/Build:{{ JENKINS_DEV_USERNAME }}"
-        - "Job/Cancel:{{ JENKINS_DEV_USERNAME }}"
         - "Job/Read:{{ JENKINS_DEV_USERNAME }}"
-        - "Job/Workspace:{{ JENKINS_DEV_USERNAME }}"
         - "Overall/Administer:{{ JENKINS_ADMIN_USERNAME }}"
         - "Overall/Read:authenticated"
-        - "Run/Replay:{{ JENKINS_DEV_USERNAME }}"
```

Comment:
Replay should be available for the 'Harvester' user.

Reply:
@khushboo-rancher - awesome :smile:  - thank you for catching that - I've updated this PR as well as
in the other-branch for testing:
- feat/2096-infrastructure-provisioning-edits